src/challenges.js

```diff
@@ -1,49 +1,112 @@
 // Desafio 1
-function compareTrue() {
-  // seu código aqui
+function compareTrue(firstBool, secondBool) {
+  return firstBool && secondBool;
 }
 
 // Desafio 2
-function calcArea() {
-  // seu código aqui
+function calcArea(base, height) {
+  return (base * height) / 2;
 }
 
 // Desafio 3
-function splitSentence() {
-  // seu código aqui
+function splitSentence(string) {
+  return string.split(' ');
 }
 
 // Desafio 4
-function concatName() {
-  // seu código aqui
+function concatName(stringArray) {
+  return `${stringArray[stringArray.length - 1]}, ${stringArray[0]}`;
 }
 
 // Desafio 5
-function footballPoints() {
-  // seu código aqui
+function footballPoints(wins, ties) {
+  return wins * 3 + ties;
 }
 
 // Desafio 6
-function highestCount() {
-  // seu código aqui
+function getHighestNumber(numberArray) {
+  let highestNumber = numberArray[0];
+
+  for (let number of numberArray) {
+    if (number > highestNumber) highestNumber = number;
+  }
+
+  return highestNumber;
+}
+
+function highestCount(numberArray) {
+  let highest = getHighestNumber(numberArray);
+  let count = 0;
+
+  for (let number of numberArray) {
+    if (number === highest) count += 1;
+  }
+
+  return count;
 }
 
 // Desafio 7
-function catAndMouse() {
-  // seu código aqui
+function catAndMouse(mouse, cat1, cat2) {
+  let firstCatDistance = 0;
+  let secondCatDistance = 0;
+
+  firstCatDistance = cat1 > mouse ? cat1 - mouse : mouse - cat1;
+  secondCatDistance = cat2 > mouse ? cat2 - mouse : mouse - cat2;
```

Comment:
Luiz, nesse trecho não seria interessante usar o `Math.abs()`? 
```suggestion
  firstCatDistance = Math.abs(cat1 - mouse);
  secondCatDistance = Math.abs(cat1 - mouse);
```

Reply:
Sim, acabei usando em outra parte e esquecendo de alterar aqui. Valeu!